src/devices/Nmea0183/Sentences/NmeaSentence.cs

```diff
@@ -86,7 +87,7 @@ public DateTimeOffset DateTime
             {
                 return _dateTime;
             }
-            protected set
+            set
```

Comment:
same question

Reply:
This one needs to be settable, because there are a few cases where the time of a message is adjusted after its creation (particularly in messages that do not transport the time)